app/src/main/java/com/bogdan/codeforceswatcher/NotificationReceiver.kt

```diff
@@ -0,0 +1,56 @@
+package com.bogdan.codeforceswatcher
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.graphics.Color
+import android.support.v4.app.NotificationCompat
+import com.bogdan.codeforceswatcher.activity.MainActivity
+
+class NotificationReceiver : BroadcastReceiver() {
+
+    override fun onReceive(context: Context?, intent: Intent?) {
+        onRefresh()
+        if (LoadUser.textNotification != "") {
+            val notificationManager = context?.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+
+            val intentReceiver = Intent(context, MainActivity::class.java)
+            intentReceiver.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP
+
+            val pIntent = PendingIntent.getActivity(context, 0, intentReceiver, PendingIntent.FLAG_UPDATE_CURRENT)
+
+            val channelId = "1234"
+
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
```

Comment:
Here was an error detected by Android Studio. It was not really the case because you have a check, but you could move it another method and add annotation to resolve an issue.